### PR TITLE
Include documentation for .navbar-static-top

### PR DESCRIPTION
**Pull request by tabone**

### CODE REVIEW

File: docs/components/navbar.md [hunk 1]
```diff
@@ -178,6 +178,14 @@ Navbars can be statically placed (their default behavior), or fixed to the top o
 </nav>
 {% endexample %}
 
+When using full width, statically placed navbars you might want to remove the rounded corners, this can be done by including `.navbar-static-top`.
```
Comment (kkirsche) [inline docs/components/navbar.md]: I think this is missing that this only applies at the top of a page, not statically placed anywhere on the page

Comment (tabone) [inline docs/components/navbar.md]: Good point however taking a look at the code (https://github.com/twbs/bootstrap/blob/v4-dev/scss/_navbar.scss#L23-L29) it seems that it doesn't modifies any position related CSS, unlike `.navbar-fixed-top`  and `.navbar-fixed-bottom` (https://github.com/twbs/bootstrap/blob/v4-dev/scss/_navbar.scss#L32-L52).

screen shot 2015-08-22 at 20 20 21

Comment (petetnt) [inline docs/components/navbar.md]: Is there any reason why this component should even have the `-top` extension? Like @tabone says, there's nothing to denote the position of the component?

**Comment by mdo** [comment]
Alternate approach in 584cfdd.

**Comment by cvrebert** [comment]
@mdo So what's the new naming scheme then?

**Comment by mdo** [comment]
How do you mean @cvrebert?

**Comment by cvrebert** [comment]
@mdo Sorry. Now that I've re-read the whole v4 navbar file, new question: Is the full-width-ness really specific to `.navbar-full`? I can't see what in the CSS suggests that.

**Comment by mdo** [comment]
Oh, yeah, no that's a good point. A regular navbar can be 100% wide, too, but it'd include rounded corners. Unsure how to do this—maybe we reverse the logic? All navbars are not rounded, unless you add `.navbar-rounded` or something?

**Pull request closed without merging**